packages/amplication-server/src/core/entity/entity.resolver.spec.ts

```diff
@@ -15,11 +15,11 @@ import { ConfigService } from "@nestjs/config";
 import { Entity } from "../../models/Entity";
 import { User } from "../../models/User";
 import { EntityField } from "../../models/EntityField";
-import {
+import type {
```

Comment:
are you sure we need the "type", because I saw in other places that we don't need it